README.md

```diff
@@ -1,110 +1,31 @@
 ![](https://raw.githubusercontent.com/auth0/wp-auth0/master/banner-1544x500.png)
 
-Wordpress Plugin for Auth0
+Login by Auth0
```

Comment:
This is for consistency from elsewhere?

Reply:
@cocojoe - That's the name on WordPress.org and in the plugin